doc/help-button.md

```diff
@@ -1,43 +1,18 @@
-### Default
+# ASAB WebUI HelpButton component
 
-Help button may have default href. You can define this default href in configuration of your application:
+Display information info for the desired screen in the modal screen.
 
-```
-let ConfigDefaults = {
-    ...
-    default_help_url: "https://github.com/TeskaLabs/asab-webui",
-    ...
-}
-```
-
-### Dynamic href
+If you wand to add this component, you need to call the `addHelpButton` method. This method will take 1 parameter:
+>- Parameter is `path`, for example (Exports/Detail, Dashboards/SomeDashboardName)
+## Example code
 
-For changing href, icon or target of Help Button you can call function (`props.app.addHelpButton()`) that will automatically add Help Button with href, icon and target defined by you. On unmount of component where this function was called it will also change button's href to default one with default icon and target if default href is specified or it will remove button from the screen if default href isn't provided.
-
-This function (`props.app.addHelpButton(href)`) requires obligatory argument `href`, it is your custom href to another page.
-
-Example:
-```
-const YourContainer = (props) => {
-    props.app.addHelpButton("https://www.your-custom-href.com");
-    return (...);
+```javascript
+const ExportsContainerDetail = (props) => {
+	props.app.addHelpButton("Dashboards/SomeDashboardName");
```

Comment:
@Aringoaway In this example, it makes no sense to me to have `ExportsContainerDetail` and path as `Dashboards/SomeDashboardName` . Please, make it more sensible.

```
export function Container(props) {
	props.app.addHelpButton("Path/to/help-content");
}
```

Also specify, where the content of the help should be stored within the Library (from the description you wrote it is not really clear). Then when merged, please ask purple-sec people to add some to the lmio common library (if needed)

Reply:
Also, please specify, if (and for what cases) should be file extension specified. But I believe only supported file extension here is `json`, am I right?